### PR TITLE
Added fonts locally

### PR DESCRIPTION
**Pull request by ChrisWeissmann**
# Issue #109 

## Description

Removed google fonts import and added them in the assets folder. Not sure this is the corret way.
Appriate any comment on if this is correct.

### CODE REVIEW

File: src/themes/default/styles/variables/_fonts.scss [hunk 1]
```diff
@@ -1,19 +1,155 @@
-@import url("https://fonts.googleapis.com/css?family=PT-Sans|PT+Sans+Caption|PT+Sans+Narrow");
+/* latin-ext */
+@font-face {
+	font-family: "PT Sans";
+	font-style: normal;
+	font-weight: 400;
```
Comment (adrianjost) [inline src/themes/default/styles/variables/_fonts.scss]: We could define the font-swap property to improve the rendering and give us more control: e.g. `font-display: swap;`

Comment (adrianjost) [reply src/themes/default/styles/variables/_fonts.scss]: Da wir hier keine präferenz haben, ist auto auch okay.

**Comment by adrianjost** [comment]
@ChrisWeissmann you can merge, if you agree with my opinion 😅

**Comment by ChrisWeissmann** [comment]
I had contact with the maintainers in Discord and from what I understand the problem is, is that the preview-head.html (what is loaded before the stories) is noth parsed through webpack and theirfore the files aren't loaded. I don't know a solution for this :/ I would also merge so that the nuxt client fonts are at least local

**Pull request merged**